lib/uniffi-zcash/src/udl/zcash_primitives/sapling/keys/outgoing_viewing_key.udl

```diff
@@ -1,3 +1,6 @@
 interface ZcashOutgoingViewingKey {
+  [Name=from_bytes, Throws=ZcashError]
+  constructor(sequence<u8> b);
```

Comment:
I have discovered (thanks to you) that we can pass this bytes in the borrowed form by using `[ByRef]` . i am unsure if that really makes a difference TBH.